src/hivpy/column_names.py

```diff
@@ -0,0 +1,21 @@
+# Column names in the Population Data-Frame
```

Comment:
This seems a good example of a module-level docstring.
```suggestion
"""Column names in the Population Data-Frame."""
```